### PR TITLE
Clean up main.go

### PR DESCRIPTION
**Pull request by jarifibrahim**
Clean up some old (unused) code

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,148 +1,42 @@
...
-	}
-	defer haltSentry()
-
 	printUserInfo()
 
 	// Create service
 	service := goa.New("golang-foo")
 
 	// Mount middleware
 	service.Use(middleware.RequestID())
```
Comment (kwk) [inline main.go:23]: Do we really need this for an example that shall just show how to start?

File: main.go [hunk 2]
```diff
@@ -1,148 +1,42 @@
...
-	// Use our own log request to inject identity id and modify other properties
 	service.Use(gzip.Middleware(9))
```
Comment (kwk) [inline main.go:24]: Do we really need this for an example that shall just show how to start?

File: main.go (hunk 1 above)
Comment (jarifibrahim) [reply main.go:23]: We don't really need it. I thought it would be okay to keep it. Shall we remove it?

File: main.go (hunk 2 above)
Comment (jarifibrahim) [reply main.go:24]: We don't really need it. I thought it would be okay to keep it. Shall we remove it?

**Pull request closed without merging**